### PR TITLE
adding an ad example

### PR DESCRIPTION
**Pull request by flashdesignory**
This ad is a css-animations based example. It supports a few different sizes that can get used in the news-sites as an ad campaign example.

https://github.com/GoogleChromeLabs/benchmark-workloads/assets/372973/b6992a6f-49e3-4dc3-b4fc-75d0d716451c

@kara

### CODE REVIEW

File: apps/ad-campaigns/ad-css-animations/dist/src/index.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        jsConfetti.addConfetti();
+    });
+
+    let animationIndex = 0;
+    let frameDelay = 2000;
+
+    function handleFrameAnimationsComplete() {
+        animationIndex++;
+        if (frames[animationIndex] === undefined) {
+            console.log("all done");
```
Comment (kara) [inline apps/ad-campaigns/ad-css-animations/dist/src/index.js:27]: Maybe remove console log?

File: apps/ad-campaigns/ad-css-animations/styles/fonts.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
+@font-face {
+    font-family: 'Poppins Regular';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Poppins Regular'), url('../assets/fonts/poppins/Poppins-Regular.woff') format('woff');
+    }
+    
+
+    @font-face {
```
Comment (kara) [inline apps/ad-campaigns/ad-css-animations/styles/fonts.css:9]: Are we using all of these font styles? Seems like we don't need to add all weights

File: apps/ad-campaigns/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Ad Campaigns
```
Comment (kara) [inline apps/ad-campaigns/README.md:1]: Nit: should we consider another folder for reusable pieces of UI? Right now, the two folders in apps are both self-contained apps in themselves. Or consider renaming "apps" => "templates"?

**Review by kara — APPROVED** [review]
LGTM with nits

File: apps/ad-campaigns/README.md (hunk 3 above)
Comment (flashdesignory) [reply apps/ad-campaigns/README.md:1]: yeah it feels weird with the ads. They don't fit nicely with the "apps" and they don't belong in "examples". 
Maybe `partials`, `components` or `ui-elements`?

Comment (flashdesignory) [reply apps/ad-campaigns/README.md:1]: I like to keep the `apps` folder for the news-sites and todomvc apps though and rather find another home for the ads

Comment (flashdesignory) [reply apps/ad-campaigns/README.md:1]: `ui-widgets` - a collection of small independent apps that can get embedded into a workload. 

These ui-widgets are functional standalone apps, that might not be beneficial to run in a benchmark by itself, but might be helpful to a benchmark, when embedding into another workload.

File: ui-widgets/ad-campaigns/ad-css-animations/index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                    viewBox="0 0 512 512">
+                    <path
+                        d="M463.5 224H472c13.3 0 24-10.7 24-24V72c0-9.7-5.8-18.5-14.8-22.2s-19.3-1.7-26.2 5.2L413.4 96.6c-87.6-86.5-228.7-86.2-315.8 1c-87.5 87.5-87.5 229.3 0 316.8s229.3 87.5 316.8 0c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0c-62.5 62.5-163.8 62.5-226.3 0s-62.5-163.8 0-226.3c62.2-62.2 162.7-62.5 225.3-1L327 183c-6.9 6.9-8.9 17.2-5.2 26.2s12.5 14.8 22.2 14.8H463.5z" />
+                </svg></button>
+        </div>
+        <div class="border"></div>
+    </div>
+</body>
+
+</html>
```
Comment (camillobruni) [inline ui-widgets/ad-campaigns/ad-css-animations/index.html:46]: minor nit about missing newlines at the end here and there.

**Review by camillobruni — APPROVED** [review]
LGTM with nit

File: apps/ad-campaigns/README.md (hunk 3 above)
Comment (kara) [reply apps/ad-campaigns/README.md]: Works for me!

Comment (flashdesignory) [reply apps/ad-campaigns/README.md]: yay!

**Pull request merged**